sonar-project.properties

```
@@ -0,0 +1,39 @@
+# Project
+sonar.organization=ngworker
+sonar.projectKey=ngworker_lumberjack-next
+sonar.projectVersion=14.0.0-beta.0
+# This is the name displayed in the SonarCloud UI.
+sonar.projectName=lumberjack-next
+# new issues will be assigned to this user each time it is not possible to
+# determine the user who is the author of the issue
+sonar.issues.defaultAssigneeLogin=LayZeeDK@github
+
+# Scan
+sonar.sourceEncoding=UTF-8
+
+# comma-separated paths to directories containing source files
+sonar.sources=apps,libs,tools
+# configure the files that should be completely ignored by the analysis
+sonar.exclusions=**/*.spec.ts
+
+# comma-separated paths to directories containing tests
+sonar.tests=apps,libs,tools
+# patterns used to include some test files and only these ones in analysis
+sonar.test.inclusions=**/*.spec.ts
+
+# configure the files that should be ignored by code coverage calculations
+sonar.coverage.exclusions=tools/scripts/**,libs/internal/**,**/jest.config.js,**/test.ts,apps/**/main.ts,apps/**/polyfills.ts
+
+# configure the files that should be ignored by duplication detection
+sonar.cpd.exclusions=**/jest.config.js,**/test.ts,apps/**/main.ts,apps/**/polyfills.ts
+
+# Reports
+# Test coverage reports
+# paths (absolute or relative) to the files with LCOV data
+# [No wildcards are allowed]
+sonar.javascript.lcov.reportPaths=coverage/apps/lumberjack-app/lcov.info,coverage/libs/internal/console-driver/test-util/lcov.info,coverage/libs/internal/test-util/lcov.info,coverage/libs/ngworker/lumberjack/lcov.info
+
+# Lint reports
+# comma-delimited list of paths to ESLint reports in JSON format
+# [No wildcards are allowed]
+sonar.eslint.reportPaths=reports/apps/lumberjack-app-e2e/lint/report.json,reports/apps/lumberjack-app/lint/report.json,reports/libs/internal/console-driver/test-util/lint/report.json,reports/libs/internal/test-util/lint/report.json,reports/libs/ngworker/lumberjack/lint/report.json
```

issue: We must have a placeholder here to generate the paths dynamically
```suggestion
sonar.eslint.reportPaths=<PLACEHOLDER>
```

---

I set the strings manually to test some stuff, but yeah I will put them back.